Apps/Database/src/Context/GatewayDbContext.cs

```diff
@@ -697,6 +697,21 @@ private void SeedEmail(ModelBuilder modelBuilder)
                     CreatedDateTime = this.DefaultSeedDate,
                     UpdatedBy = UserId.DefaultUser,
                     UpdatedDateTime = this.DefaultSeedDate,
+                },
+                new EmailTemplate
+                {
+                    Id = Guid.Parse("75c79b3e-1a61-403b-82ee-fddcda7144af"),
+                    Name = "AdminFeedback",
+                    From = "HG_Donotreply@gov.bc.ca",
+                    Subject = "Health Gateway Feedback Received",
```

Comment:
Nit: would suggest renaming Subject to "Health Gateway Feedback" since these emails will often be replied to and `RE: Health Gateway Feedback` would make more sense for a user than `RE: Health Gateway Feedback Received`.

Reply:
We can update this directly in the DB post deploy.